src/GameKit/GameKit.cs

```diff
@@ -128,8 +128,37 @@ public enum GKError : nint {
 #else
 		InvitationsDisabled = 25, // iOS 7.0
 		PlayerPhotoFailure = 26,  // iOS 8.0
-		UbiquityContainerUnavailable = 27 // iOS 8.0
+		UbiquityContainerUnavailable = 27, // iOS 8.0
 #endif
+		MatchNotConnected = 28,
+		GameSessionRequestInvalid = 29,
+	}
+
+	[Native]
```

Comment:
while not critical (not code, just data) it's nicer to have an availability attribute on new enums